server/athenian/api/align/goals/dbaccess.py

```diff
@@ -192,6 +194,61 @@ async def delete_empty_goals(account: int, sdb_conn: DatabaseLike) -> None:
     await sdb_conn.execute(delete_stmt)
 
 
+async def get_goal_template_from_db(template_id: int, sdb: DatabaseLike) -> Row:
+    """Return a GoalTemplate."""
+    stmt = sa.select(GoalTemplate).where(GoalTemplate.id == template_id)
+    template = await sdb.fetch_one(stmt)
+    if template is None:
+        raise GoalTemplateNotFoundError(template_id)
+    return template
+
+
+async def get_goal_templates_from_db(account: int, sdb: DatabaseLike) -> Row:
+    """Return all account GoalTemplate-s, ordered by id."""
+    stmt = (
+        sa.select(GoalTemplate).where(GoalTemplate.account_id == account).order_by(GoalTemplate.id)
+    )
+    return await sdb.fetch_all(stmt)
+
+
+async def goal_template_exists(template_id: int, account: int, sdb: DatabaseLike) -> bool:
+    """Return true if the give template exists for the account."""
+    stmt = sa.select(1).where(GoalTemplate.id == template_id, GoalTemplate.account_id == account)
+    return (await sdb.fetch_val(stmt)) is not None
+
+
+async def account_goals_are_migrated(account: int, sdb: DatabaseLike) -> bool:
+    """Find find whether account has already been migrated to DB templates."""
+    # an account is migrated if the default templates have been created for the account
+    # and account hasn't any goal pointing to old hard-coded templates
+    # TODO: drop this function after migration is complete for everyone
```

Comment:
```suggestion
    # TODO(gaetano): drop this function after migration is complete for everyone
```
(google style TODO so that's faster to know who to ask about the TODO)